modules/social_features/social_event/modules/social_event_managers/src/Plugin/Action/SocialEventManagersSendEmail.php

```diff
@@ -97,7 +97,8 @@ public function access($object, AccountInterface $account = NULL, $return_as_obj
     $access = AccessResult::allowedIf($object instanceof EventEnrollmentInterface);
 
     if ($object instanceof EventEnrollmentInterface) {
-      $access = $object->access('delete', $account, TRUE);
+      // All users with the following access permission should be allowed.
+      $access = AccessResult::allowedIfHasPermission($account, 'manage everything enrollments');
```

Comment:
@navneet0693 We check the same permission later in the [function](https://github.com/goalgorilla/open_social/blob/ede8602c63fabdc633aa0f90d691841ec76274f4/modules/social_features/social_event/social_event.module#L135) `social_event_manager_or_organizer`. Is it necessary here?

Reply:
@Ressinel This is the default fallback permission, even if we check the same permission it’s only going to be checked if we actually hit the if statement, if we don’t we at least have this default fallback permission.